### PR TITLE
Revert api doc for 1.19 about PortSpecs removal

### PR DESCRIPTION
**Pull request by runcom**
ping @thaJeztah 
fixes api docs for 1.19 pulled into master, now in 1.20

Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 1]
```diff
@@ -333,6 +333,7 @@ Return low-level information on the container `id`
 			"NetworkDisabled": false,
 			"OnBuild": null,
 			"OpenStdin": false,
+            "PortSpecs": null,
```
Comment (runcom) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: @thaJeztah do not ask me why this is displayed like this :smile: even `git diff` command do this but rendering is perfect

Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: @runcom looks like the lines above your change are using tabs :'(

In this case, perhaps you can change it to use a tab as well, then we can fix it up later to convert all tabs to spaces in a separate PR.

You can check that by selecting/highlighting the whitespace with your mouse and dragging (well, something like that)

Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: @runcom I see the other parts *do* use spaces, so I would also be okay with changing the tabs to spaces in this block

Comment (runcom) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: yahi you're right! I spent 5 minutes looking for tabs but didn't look at previous line lol (and I don't highlight text in vim with mouse :/)

File: docs/sources/reference/api/docker_remote_api_v1.19.md [hunk 2]
```diff
@@ -332,7 +332,8 @@ Return low-level information on the container `id`
 			"MacAddress": "",
 			"NetworkDisabled": false,
 			"OnBuild": null,
-			"OpenStdin": false,
+            "OpenStdin": false,
+            "PortSpecs": null,
```
Comment (runcom) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: oh fml ahhaah

**Comment by thaJeztah** [comment]
For reference: these changes were made as part of https://github.com/docker/docker/pull/13446, but that PR won't be included in docker 1.7, so the changes had to be moved from API  v1.19 to 1.20

File: docs/sources/reference/api/docker_remote_api_v1.19.md (hunk 2 above)
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: @runcom :'( sorry, I noticed too late

Comment (runcom) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: fixed fixed :)

File: docs/sources/reference/api/docker_remote_api_v1.19.md (hunk 1 above)
Comment (thaJeztah) [inline docs/sources/reference/api/docker_remote_api_v1.19.md]: Hehe, but the trick works on GitHub :)

**Comment by thaJeztah** [comment]
Thanks! LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**